src/Symfony/Component/HttpKernel/Tests/Controller/ControllerResolverTest.php

```diff
@@ -194,7 +196,7 @@ public function getUndefinedControllers()
         );
     }
 
-    protected function createControllerResolver(LoggerInterface $logger = null): ControllerResolverInterface
+    protected function createControllerResolver(LoggerInterface $logger = null, ContainerInterface $container = null, ControllerNameParser $parser = null): ControllerResolverInterface
```

Comment:
The HttpKernel does not know about ControllerNameParser. So it needs a different solution.